packages/drivers/odsp-driver/src/localOdspDocumentService.ts

```diff
@@ -0,0 +1,54 @@
+/*!
```

Comment:
Should we add these 3 new files starting with "local" into a separate folder inside odsp driver/src/..?